.github/workflows/build-release-binaries.yml

```diff
@@ -0,0 +1,98 @@
+name: Release Bitshares-ui
+
+on:
+    push:
+        tags:
+            - "*.*.*"
+            # branches: [testaction]
+
+jobs:
+    build:
+        runs-on: macOS-latest
+
+        strategy:
+            matrix:
+                node-version: [16.13.x]
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - run: echo "PackageVersion=$(cat package.json | sed 's/,/\n/g' | grep '\"version\":' | sed 's/:/\n/g' | sed '1d' | sed 's/ //g' | sed 's/\"//g' | sed 's/}//g')" >> $GITHUB_ENV
```

Comment:
To be honest, these long lines look ugly when viewing Github.com in a web browser.

Reply:
These marks are the most basic, we can add new ones, and we can add them according to the manager's ideas.

Reply:
It can be optimized here, such as using jq, but I don't want to use too many external programs.

Reply:
Commands can be simply wrapped with the `\` sign and etc. Actually, since you have `(` in the command, it supports multiple lines by itself already. E.G.
```
- run: |
    echo "PackageVersion=$(cat package.json | sed 's/,/\n/g' | grep '\"version\":' |
        sed 's/:/\n/g' | sed '1d' | sed 's/ //g' | sed 's/\"//g' | sed 's/}//g')" \
        >> $GITHUB_ENV
```
Just my 2 cents.

Reply:
Note: this is not the only long line in the pull request. It's best to wrap them all.